AutomatedTesting/Gem/PythonTests/scripting/VariableManager_ExposeVarsToComponent.py

```diff
@@ -0,0 +1,93 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+# stub for when save file code is available
+ENTITY_NAME = "SC_Component_Variable_Entity"
+COMPONENT_LIST = ["Script Canvas"]
+VARIABLE_NAME = "Test Boolean"
+
+def VariableManager_ExposeVarsToComponent():
+    """
+    Summary:
+     Test for verifying variables exposed to component can be accessed and modified in Editor
+
+    Expected Behavior:
+     Each variable type can be created and deleted in variable manager.
+
+    Test Steps:
+     1) Open Script Canvas window (Tools > Script Canvas)
+     2) Get the SC window object
+     3) Open Variable Manager if not opened already
+     4) Create Graph
+     5) Create variable of each type and change their initial value source to from component
+     6) Save the file
+     7) Create an entity w/ component in the O3DE editor and attach the file
+     8) Verify the new variables are accessible and can be modified.
+
+    Note:
+     - Any passed and failed tests are written to the Editor.log file.
+            Parsing the file or running a log_monitor are required to observe the test results.
+
+    :return: None
+    """
+
+    # Preconditions
+    from editor_python_test_tools.QtPyO3DEEditor import QtPyO3DEEditor
+    from scripting_utils.script_canvas_component import ScriptCanvasComponent, VariableState
+    from editor_python_test_tools.QtPyScriptCanvasNodeInspector import BooleanCheckBoxValues
+    import azlmbr.legacy.general as general
+    from consts.scripting import (SCRIPT_CANVAS_TEST_FILE_PATH)
+    from editor_python_test_tools.editor_entity_utils import EditorEntity
+    from editor_python_test_tools.utils import TestHelper
+    import azlmbr.math as math
+
+    general.idle_enable(True)
+
+    # 1) Open Script Canvas window
+    qtpy_o3de_editor = QtPyO3DEEditor()
+    TestHelper.open_level("", "Base")
+    sc_editor = qtpy_o3de_editor.open_script_canvas()
```

Comment:
NIT: 

Let's reorder this.

Open the level first, add white space, then get the O3DE editor reference then open the sc editor.

This will help with readability and logical understanding during troubleshooting.